Java/syntax_test_java.java

```diff
@@ -16,6 +16,9 @@
 //            ^ meta.import.java support.function.import.java
 //                               ^ meta.import.java punctuation.terminator.java
 
+import static a.b.Class.CONSTANT;
```

Comment:
Let's add some tests here for `punctuation.accessor`

Reply:
Done. It only matches the last dot due to the limitations of the pattern.